apis/sfn/v1alpha1/zz_types.go

```diff
@@ -0,0 +1,157 @@
+/*
+Copyright 2020 The Crossplane Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+// Code generated by ack-generate. DO NOT EDIT.
+
+package v1alpha1
+
+import (
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+)
+
+type ActivityListItem struct {
+	ActivityARN *string `json:"activityARN,omitempty"`
+
+	CreationDate *metav1.Time `json:"creationDate,omitempty"`
+
+	Name *string `json:"name,omitempty"`
+}
```

Comment:
None of the structs in this file have comments for documentation, do they need documentation (i.e., will any of these be exposed to users via CRDs)?

Reply:
Comments are available in ACK only for certain fields and I couldn't find a good way to inject them properly. For now, I opted to get back to ACK to fix them later on.

Reply:
Ah okay, I see, thanks for clarifying